cmd/apps.go

```diff
@@ -104,11 +106,19 @@ func AppInfo(appID string) error {
 		return err
 	}
 
+	url, err := appURL(s, appID)
+	if checkAPICompatibility(s.Client, err) != nil {
```

Comment:
doesn't appURL already do an API compatibility check on line 294? Is there significance to checking it twice?